### DISCUSSION TITLE
Arduino_GFX commands

### DISCUSSION THREAD

**Discussion by Salam797** [moononournation/Arduino_GFX#440] [Q&A]
Hi Moononournation, could you please provide me with all of the commands that are used with your GFX library? I find it very confusing and difficult sometimes because some of the commands work and some are similar to other libraries but do not work.  Therefore your assistance with this matter would be greatly appreciated.

Thanks

**Comment by Salam797** [comment]
> Hi Moononournation, could you please provide me with all of the commands that are used with your GFX library? I find it very confusing and difficult sometimes because some of the commands work and some are similar to other libraries but do not work.  Therefore your assistance with this matter would be greatly appreciated.
> 
> Thanks

Hi Moononournation, please could you please assist me? My display won't work except using your library.

**Reply by moononournation** [reply]
Please study the examples, especially PDQgraphicstest.ino
